internal/osutil/exitCode.go

```diff
@@ -6,5 +6,8 @@ type ExitCode int
 // ExitCode constants.
 const (
 	ExitSuccess ExitCode = iota
-	ExitFailure
+	ExitInternalFailure
```

Comment:
I agree with keeping it similar to other linters; I just wish there was a "standard" 😆 . I can easily change the error codes to be another way around; I have no preference. Let me know what you prefer?

The logic for the current setup was based on the fact that I wanted to make sure that runtime errors were 2, as that is the default go behaviour, and so reduced the complexity. However, adding a `defer func(){recover()...}()` is not much effort.
If you would like to go for the simplest option, I suggest:
- `1`: lint
- `2`: anything else. One could argue that a parsing issue is an internal issue.

An alternative approach, which would allow you to have internal issues as a separate exit code, could be to make internal be `255`. Runtime could be `254`. This way, low codes could be file issues (lint/parse) versus high codes for tool issues (arguments, runtime).
This would give something like:
- `1` lint
- `2` parse
- `254`: runtime
- `255`: internal

Of these 2 options, I personally prefer the first, simplest, option. Let me know what you think!

Reply:
> Of these 2 options, I personally prefer the first, simplest, option. Let me know what you think!

@PaulSonOfLars Thank you for your clear explanation!
I also prefer the first option.